code/src/index.css

```diff
@@ -11,3 +12,95 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+.header {
+  font-size: 50px;
+  padding: 20px;
+  text-align: center;
+}
+
+.header p {
+  font-size: 26px;
+  color:#FFADAD;
+}
+
+.form-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  width: 80%;
+  max-width: 500px;
+  margin: 0 auto 40px auto;
+  padding: 20px;
+  background-color: #f3f1f1;
+  border: 2px solid #000;
+  box-shadow: 6px 6px #000;
+}
+
+.thought-list {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: left;
+}
+
+.thought-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  width: 80%;
+  max-width: 500px;
+  margin: auto;
+  margin-bottom: 40px;
+  padding: 20px;
+  background-color: #fff;
+  border: 2px solid #000;
+  box-shadow: 6px 6px #000;
+  word-wrap: break-word;
+}
```

Comment:
Since you are using Flexbox on most of these classes you perhaps could have put them in one declaration